### PR TITLE
1. created a database using knex module 2. added progress notes and u…

### PR DESCRIPTION
**Pull request by majid76**
…pdated my progress there 3.added .gitignore file

### CODE REVIEW

File: database.js
Comment (AhmedShab) [inline database.js]: I would strongly suggest that you should create `.env` file and add it to your ignore list. All you would need then is requiring the variables you need.

Have a looks at this npm, it's the best way to secure your configuration:

https://github.com/motdotla/dotenv

**Comment by AhmedShab** [comment]
@majid76 Awesome stuff mate. I will merge it when you're done fixing what I commented on. Good work 🚀 🏆 😄

**Review by AhmedShab — CHANGES_REQUESTED** [review]
@majid76 Awesome stuff mate. I will merge it when you're done fixing what I commented on. Good work 🚀 🏆 😄

**Comment by AhmedShab** [comment]
I would also suggest naming your branch to a specific task. For example, `databaseSetup`.

**Comment by majid76** [comment]
will do Sirrrrr 😁

**Comment by AhmedShab** [comment]
@majid76 Don't worry about this branch name for now. You can do it after you're done with this merge (your next branch 👍 )

**Comment by majid76** [comment]
ok bro.

**Pull request closed without merging**
